src/check/parse/tokenize.zig

```diff
@@ -435,9 +449,15 @@ pub const Cursor = struct {
                 }
             } else if (b == '#') {
                 self.pos += 1;
+                const comment_start = self.pos;
```

Comment:
I would rather make this part of the return value (eg in a struct) than stuff this in a field on self

Reply:
Could you explain why?

Reply:
I was thinking of removing the u16 for the indent completely - since it's no longer used (or needed).

Reply:
That makes sense to me!